src/views/catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeSource/VolumeSource.tsx

```diff
@@ -51,26 +49,18 @@ const VolumeSource: FC<VolumeSourceProps> = ({
           )
         }
       />
-      <Checkbox
-        id="clone-pvc-checkbox"
-        isChecked
-        isDisabled
-        label={
-          <Split hasGutter>
-            <SplitItem>{t('Clone existing PVC')}</SplitItem>
-            <SplitItem onClick={(e) => e.preventDefault()}>
-              <Popover
-                position="right"
-                bodyContent={t(
-                  'This will create a cloned copy of the PVC in the destination project.',
-                )}
-              >
-                <HelpIcon className="icon-size-small" />
-              </Popover>
-            </SplitItem>
-          </Split>
-        }
-      />
+
+      <Split hasGutter>
+        <SplitItem>
+          <Checkbox id="clone-pvc-checkbox" isChecked isDisabled label={t('Clone existing PVC')} />
+        </SplitItem>
+        <SplitItem>
+          <HelpTextIcon
+            bodyContent={t('This will create a cloned copy of the PVC in the destination project.')}
```

Comment:
Suggestion: "This creates a cloned copy ..." 
IBM style guide recommends using present tense instead of future tense, whenever possible.